assignment.js

```diff
@@ -101,13 +119,26 @@ Use the following test cases to confirm your program meets the success criteria
    buzz
 
 ************/
-console.log('Problem 3:')
-
 // Add your code below this line
+console.log('Problem 3:')
 
+let whenFizz = "fizz"
+let whenBuzz = "buzz"
+let whenFizzBuzz = "fizzbuzz"
+
+for (let i = 0; i <= 100; i++) {
+  if (i%3 === 0 && i%5 === 0) {
```

Comment:
@kyleligman for readability purposes you should add spacing when using arithmetic operators:

`i % 3 === 0` instead of `i%3 === 0`